### ISSUE TITLE
Bug on Browser Engine: `browser.newContext: Target page, context or browser has been closed.`

### ISSUE THREAD

**Issue by calebcall** [louislam/uptime-kuma#4373]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

I get this error with the browser engine checks: 

```log
browser.newContext: Target page, context or browser has been closed. 
```

The Checks using the Browser Engine work great for an initial period of time and then all of the checks using the browser engine start to fail at the same time and never recover.

### 👟 Reproduction steps

Configure Browser Engine monitor

### 👀 Expected behavior

To continue to work

### 😓 Actual Behavior

return the error : 

```log
browser.newContext: Target page, context or browser has been closed
```

### 🐻 Uptime-Kuma Version

1.23.11

### 💻 Operating System and Arch

Debian 12.4

### 🌐 Browser

Brave v1.61.116

### 🐋 Docker Version

K8s 1.27.7

**Comment by CommanderStorm** [comment]
- What exactly is returning this log message? the browser console or the node/docker log?
- To be able to reproduce this: what website are you monitoring? Is this reproducible?

**Comment by calebcall** [comment]
I'm seeing this error in Kuma.  The services are internal services (Jellyseerr, Radarr, Sonarr, Sabnzbd, and Scrypted)

<img width="737" alt="Screenshot 2024-01-15 at 6 41 20 PM" src="https://github.com/louislam/uptime-kuma/assets/1132262/0976eeed-a8ba-40cb-8a7c-ccae30bc2148">

This shot shows they all went down in the same check period.
<img width="973" alt="Screenshot 2024-01-15 at 6 45 40 PM" src="https://github.com/louislam/uptime-kuma/assets/1132262/6966c138-bd1b-4481-87b9-7b3e38819e3f">

Prior to that, they had all been up for about 6.5 hours when they all failed in the same way.

**Comment by CommanderStorm** [comment]
Thanks ^^
We might have a race condition in the playwright code as in https://stackoverflow.com/questions/61933492/playwright-error-target-closed-after-navigation

Given that this does not seem very widespread/reprducible I am not going to prioritse reproducing/debugging this (I am currently quite under the water Uni-wise).
If somebody else wants to give this a shot, here is our contribution guide: https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md

**Comment by chakflying** [comment]
Added a simple fix in #4417 but I'm not sure if this is the root cause, you can help by testing it if you're interested.

https://github.com/louislam/uptime-kuma/wiki/Test-Pull-Requests

**Comment by CommanderStorm** [comment]
@calebcall
I think this issue was fixed by https://github.com/louislam/uptime-kuma/pull/4417
Given that we cannot reproduce the issue, testing said PR would be greatly appreaciated ^^

I am going to assume that said PR works and does not contain bugs => closing as resolved
